README.md

```diff
@@ -1,37 +1,25 @@
-# Docs
+# 🔖Docs
 
 [![Actions Status](https://github.com/woodlike/docs/workflows/CI/badge.svg)](https://github.com/woodlike/docs/actions)
 
-> Turn your Gatsby project into visual and interactive technical documentation.
+## Description
 
-## Motivation
+Docs is a Gatsby theme that programmatically generates visually polished and interactive technical documentation using MDX and Babel.
 
-### Development Requirements
+## 📚Documentation
 
-🚀 Node.js >=10
-🌲 Git
-🐈 yarn >= 1.12
+WIP
```

Comment:
What do you want to add here?

Reply:
I would like to include a link to the documentation page, which currently does not exist. I included that information next to the WIP. Do you think that is enough?